x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/monitor_list_drawer/__tests__/most_recent_error.test.tsx

```diff
@@ -21,12 +22,15 @@ describe('MostRecentError component', () => {
     };
     monitorDetails = {
       monitorId: 'bad-ssl',
-      timestamp: '2019-11-30T01:57:37.792Z',
+      timestamp: moment()
```

Comment:
I'm ok with this for now as it will likely resolve CI failures, but ideally we would mock `moment` itself to simply return some hardcoded value. We aren't testing `moment`'s ability to render correct relative date strings - we are just testing that our component renders the output of the moment call.

Reply:
@shahzad31 would you have an objection to doing something like:

```ts
import moment from 'moment';

beforeAll(() => {
  moment.prototype.fromNow = jest.fn(() => 'expected date value');
});
```

This way the snapshot will contain the same mocked value no matter what time zone or how long in the future the test is ran.

Reply:
yeah it makes sense as well

Reply:
Ok cool, let's go with that. In the meantime I think the Kibana Team will skip this test, and then we can rebase and un-skip it as part of the patch.

Reply:
Test is skipped in https://github.com/elastic/kibana/commit/05fd394325836abb0abcf206f3d5f422243b0c55, so let's rebase, add those changes, and WFG on this and then we can merge it.